index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for react-native-snackbar 0.3.2
```

Comment:
Yea it needs to be versioned in case you update and add features to the main library. So people would know if the types and library are on the same version